Maker-Agents-Suit-Module/OSMAgent/src/time.tracker.ts

```diff
@@ -0,0 +1,43 @@
+export default class TimeTracker {
+  hour: number; // keeps track of the hour
+  functionWasCalled: boolean = false;
+  findingReported: boolean = false;
+
+  constructor() {
+    this.hour = -1;
+  }
```

Comment:
Why are not all the variables initialized in the same way? Any special reason for using the constructor only for the `hour` field?